rust/src/encoding/base32.rs

```diff
@@ -0,0 +1,244 @@
+use std::ffi::{c_char, CStr, CString};
```

Comment:
can you setup danger? and check license header

Reply:
Will do in a separate PR, note that will probably smth else than danger, as danger work only for some programming languages